src/sentry/sentry_metrics/consumers/indexer/processing.py

```diff
@@ -191,14 +192,45 @@ def process_messages(
             used_tags.add(metric_name)
 
             new_tags: MutableMapping[str, int] = {}
+            dropped_string = False
             try:
                 for k, v in tags.items():
                     used_tags.update({k, v})
-                    new_tags[str(mapping[org_id][k])] = mapping[org_id][v]
+                    new_k = mapping[org_id][k]
+                    new_v = mapping[org_id][v]
+                    new_tags[str(new_k)] = new_v
+
+                    if new_k is None:
+                        fetch_type_ext = get_path(bulk_record_meta, k, 2)
+                        logger.error(
+                            "process_messages.dropped_string",
+                            extra={
+                                "is_global_quota": fetch_type_ext
+                                and fetch_type_ext.is_global_quota,
+                                "org_batch_size": len(mapping[org_id]),
+                            },
+                        )
+                        dropped_string = True
+
+                    if new_v is None:
+                        fetch_type_ext = get_path(bulk_record_meta, v, 2)
+                        logger.error(
+                            "process_messages.dropped_string",
+                            extra={
+                                "is_global_quota": fetch_type_ext
+                                and fetch_type_ext.is_global_quota,
+                                "org_batch_size": len(mapping[org_id]),
+                            },
+                        )
```

Comment:
Maybe for a follow up. I would log once per message or we risk to impact the indexer throughput with the logging work in order not to impact it with DB writes. 
Since you can have multiple quotas exceeded per message, you can just list them in the extra data. Which quota we exceeded per string is not an interesting information. Which quotas we exceeded in a message is interesting information as we would use it to troubleshoot. 
If we exceed global quota we have a scale problem. If we exceeded org quota we have an abuser.

Reply:
I updated the log messages to address this, since I felt it was easy enough to do.